zeus/client.py

```diff
@@ -139,7 +139,7 @@ def get_poll_info(url):
 
     try:
         parsed = urlparse(html.split('<a id="booth-link"')[1].split('href="')[1].split('"')[0])
-    except:
+    except ValueError:
```

Comment:
This can fail for many ways and `ValueError` is just one of them. How about just removing the try...except block? If the `urlparse(...)` line fails, just let the exception go further.

Reply:
I think that's why this except clause was catching anything, printing some data and then - raising an exception.
Do we need to print content of `html` here?